# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-06-06 14:58-0300\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: api_app/apps.py:184
msgid "Lista de agentes activos de un Grupo"
msgstr ""

#: api_app/apps.py:186
msgid "Campañas relacionadas a un supervisor"
msgstr ""

#: api_app/apps.py:188
msgid "Opciones de calificacion de una campaña (id campaña externo)"
msgstr ""

#: api_app/apps.py:191
msgid "Opciones de calificacion de una campaña (id campaña oml)"
msgstr ""

#: api_app/apps.py:194
msgid "Ver/editar calificación cliente"
msgstr ""

#: api_app/apps.py:196
msgid "Crear calificacion cliente"
msgstr ""

#: api_app/apps.py:198
msgid "Crear contacto"
msgstr ""

#: api_app/apps.py:200
msgid "Metadata de la base de datos de una campaña"
msgstr ""

#: api_app/apps.py:202
msgid "Crear nuevo rol"
msgstr ""

#: api_app/apps.py:204
msgid "Eliminar rol"
msgstr ""

#: api_app/apps.py:206
msgid "Actualizar permisos de Rol"
msgstr ""

#: api_app/apps.py:208
msgid "Devuelve información de los agentes en el sistema"
msgstr ""

#: api_app/apps.py:211
msgid "Reporte de llamadas entrantes de supervisión."
msgstr ""

#: api_app/apps.py:213
msgid "Reporte de llamadas salientes no dialer de supervisión."
msgstr ""

#: api_app/apps.py:217
msgid "Ejecuta acciones de supervisión sobre agente (Deslogueo, pausas, etc..)"
msgstr ""

#: api_app/apps.py:220
msgid "Cantidades por tipos de llamada en una campaña"
msgstr ""

#: api_app/apps.py:223
msgid "Cantidades por tipo de calificación en una campaña"
msgstr ""

#: api_app/apps.py:226
msgid "Reasignar una Agenda a otro Agente"
msgstr ""

#: api_app/apps.py:228
msgid "Información de contacto de una  Agenda"
msgstr ""

#: api_app/apps.py:230
msgid "Exportar reporte de contactados de una campaña a csv"
msgstr ""

#: api_app/apps.py:233
msgid "Exportar reporte de calificados de una campaña a csv"
msgstr ""

#: api_app/apps.py:236
msgid "Exportar reporte de no atendidos de una campaña a csv"
msgstr ""

#: api_app/apps.py:239 api_app/apps.py:242
msgid "Exportar reporte de calificaciones de una campaña a csv"
msgstr ""

#: api_app/apps.py:245
msgid "API para exportar resultados de base contactados a csv"
msgstr ""

#: api_app/apps.py:248
msgid "Contactos de una campaña"
msgstr ""

#: api_app/apps.py:250
msgid "Ejecuta un click 2 call"
msgstr ""

#: api_app/apps.py:252
msgid "Ejecuta el login del agente en Asterisk."
msgstr ""

#: api_app/apps.py:254
msgid "Ejecuta el logout del agente en Asterisk"
msgstr ""

#: api_app/apps.py:256
msgid "Logout del agente en OML"
msgstr ""

#: api_app/apps.py:258
msgid "Pone al agente en una pausa"
msgstr ""

#: api_app/apps.py:260
msgid "Saca al agente de una pausa"
msgstr ""

#: api_app/apps.py:262
msgid "Establece el estado Ringing del Agente"
msgstr ""

#: api_app/apps.py:264
msgid "Establece el estado Reject del Agente"
msgstr ""

#: api_app/apps.py:266
msgid "Devuelve credenciales SIP de un agente"
msgstr ""

#: api_app/apps.py:268
msgid "Establece el estado de revisión de una Auditoría"
msgstr ""

#: api_app/apps.py:271
msgid "Almacena en la base de datos los contactos subidos en archivo csv"
msgstr ""

#: api_app/apps.py:274
msgid "Detecta si una llamada esta calificada."
msgstr ""

#: api_app/apps.py:277
msgid "Reenvía la llave de la instancia registrada por email"
msgstr ""

#: api_app/apps.py:279
msgid "Retorna el archivo de grabación especificado"
msgstr ""

#: api_app/apps.py:282
msgid "Retorna Zip con archivos de grabación especificados"
msgstr ""

#: api_app/apps.py:285
msgid "Devuelve los contactos asignados de una campaña preview"
msgstr ""

#: api_app/apps.py:288
msgid "Loggea el evento hold o unhold"
msgstr ""

#: api_app/apps.py:291
msgid "Lista audios de asterisk"
msgstr ""

#: api_app/apps.py:294
msgid "Lista de grupos de agentes"
msgstr ""

#: api_app/apps.py:297
msgid "Lista de Agentes"
msgstr ""

#: api_app/apps.py:300
msgid "Dashboard de supervision"
msgstr ""

#: api_app/apps.py:303
msgid "Auditoría a supervisores"
msgstr ""

#: api_app/apps.py:306
msgid "Listas de nombres de columnas para Base de datos de Contactos"
msgstr ""

#: api_app/apps.py:309
msgid "Reinicia servicio Wombat Dialer"
msgstr ""

#: api_app/apps.py:312
msgid "Estado del servicio Wombat Dialer"
msgstr ""

#: api_app/apps.py:315
msgid "Lista de agentes por campaña"
msgstr ""

#: api_app/apps.py:318
msgid "Actualiza los agentes de una campaña"
msgstr ""

#: api_app/apps.py:321
msgid "Obtiene todos los agentes activos"
msgstr ""

#: api_app/apps.py:324
msgid "Obtiene las pausas que no estan eliminadas"
msgstr ""

#: api_app/apps.py:327
msgid "Lista los conjuntos de pausas"
msgstr ""

#: api_app/apps.py:330
msgid "Obtiene el detalle de un conjunto de pausas"
msgstr ""

#: api_app/apps.py:333
msgid "Crea un conjunto de pausas"
msgstr ""

#: api_app/apps.py:336
msgid "Actualiza un conjunto de pausas"
msgstr ""

#: api_app/apps.py:339
msgid "Elimina un conjunto de pausas"
msgstr ""

#: api_app/apps.py:342
msgid "Crea una configuracion de pausa"
msgstr ""

#: api_app/apps.py:345
msgid "Actualiza una configuracion de pausa"
msgstr ""

#: api_app/apps.py:348
msgid "Elimina una configuracion de pausa"
msgstr ""

#: api_app/apps.py:351
msgid "Lista los sitios externos"
msgstr ""

#: api_app/apps.py:354
msgid "Obtiene el detalle de un sitio externo"
msgstr ""

#: api_app/apps.py:357
msgid "Crea un sitio externo"
msgstr ""

#: api_app/apps.py:360
msgid "Actualiza un sitio externo"
msgstr ""

#: api_app/apps.py:363
msgid "Elimina un sitio externo"
msgstr ""

#: api_app/apps.py:366
msgid "Oculta un sitio externo"
msgstr ""

#: api_app/apps.py:369
msgid "Desoculta un sitio externo"
msgstr ""

#: api_app/apps.py:372
msgid "Lista las calificaciones"
msgstr ""

#: api_app/apps.py:375
msgid "Crea una calificacion"
msgstr ""

#: api_app/apps.py:378
msgid "Actualiza una calificacion"
msgstr ""

#: api_app/apps.py:381
msgid "Elimina una calificacion"
msgstr ""

#: api_app/apps.py:384
msgid "Obtiene detalle de una calificacion"
msgstr ""

#: api_app/forms.py:53 api_app/forms.py:98
#: api_app/tests/tests_api_click2call.py:136
#: api_app/tests/tests_api_click2call.py:208
msgid "El agente no participa en la campaña."
msgstr ""

#: api_app/forms.py:62 api_app/tests/tests_api_click2call.py:144
msgid "El contacto no corresponde a la campaña."
msgstr ""

#: api_app/forms.py:100 api_app/tests/tests_api_click2call.py:174
#: api_app/tests/tests_api_click2call.py:196
msgid "El agente no corresponde al sistema externo."
msgstr ""

#: api_app/services/base_datos_contacto_service.py:45
msgid "Ya existe una base de datos de contactos con ese nombre"
msgstr ""

#: api_app/services/base_datos_contacto_service.py:49
msgid ""
"El archivo especificado para realizar la importación de contactos no es "
"válido."
msgstr ""

#: api_app/services/base_datos_contacto_service.py:54
#: api_app/tests/tests_api_subir_base_contactos.py:204
#: api_app/views/administrador.py:195
msgid "El archivo a procesar tiene nombres de columnas repetidos."
msgstr ""

#: api_app/services/base_datos_contacto_service.py:128
msgid "Archivo excede máximo de filas permitidas"
msgstr ""

#: api_app/services/base_datos_contacto_service.py:132
msgid "Error al parsear el archivo csv"
msgstr ""

#: api_app/tests/campaign/agents/tests_add_agents_to_campaign.py:90
#: api_app/tests/campaign/agents/tests_add_agents_to_campaign.py:100
#: api_app/views/supervisor.py:923 api_app/views/supervisor.py:1085
msgid "Se obtuvieron los agentes de forma exitosa"
msgstr ""

#: api_app/tests/campaign/agents/tests_add_agents_to_campaign.py:119
#: api_app/views/supervisor.py:1015
msgid "Se agregaron los agentes de forma exitosa a la campaña"
msgstr ""

#: api_app/tests/pause_sets/tests_configuraciones_de_pausas.py:70
#: api_app/views/supervisor.py:1304
msgid "Se creo la configuracion de pausa de forma exitosa"
msgstr ""

#: api_app/tests/pause_sets/tests_configuraciones_de_pausas.py:88
#: api_app/views/supervisor.py:1384
msgid "Se actualizo la configuracion de pausa de forma exitosa"
msgstr ""

#: api_app/tests/pause_sets/tests_configuraciones_de_pausas.py:106
#: api_app/views/supervisor.py:1351
msgid "Se elimino la configuracion de pausa de forma exitosa"
msgstr ""

#: api_app/tests/pause_sets/tests_configuraciones_de_pausas.py:125
#: api_app/views/supervisor.py:1311
msgid "El timeout debe ser mayor a cero"
msgstr ""

#: api_app/tests/pause_sets/tests_configuraciones_de_pausas.py:137
#: api_app/views/supervisor.py:1359
msgid "No puedes dejar a un Conjunto de Pausas vacio"
msgstr ""

#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:67
#: api_app/views/supervisor.py:1142
msgid "Se obtuvieron los conjuntos de pausas de forma exitosa"
msgstr ""

#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:89
#: api_app/views/supervisor.py:1200
msgid "Se creo el conjunto de pausa de forma exitosa"
msgstr ""

#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:110
#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:161
#: api_app/views/supervisor.py:1205 api_app/views/supervisor.py:1241
msgid "El nombre del conjunto es requerido"
msgstr ""

#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:126
#: api_app/views/supervisor.py:1209
msgid "Debe existir al menos una pausa en el conjunto"
msgstr ""

#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:144
#: api_app/views/supervisor.py:1237
msgid "Se actualizo el conjunto de pausa de forma exitosa"
msgstr ""

#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:173
#: api_app/views/supervisor.py:1270
msgid "Se elimino el conjunto de pausa de forma exitosa"
msgstr ""

#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:187
#: api_app/views/supervisor.py:1277
msgid ""
"No puedes borrar un conjunto de pausas que esta asignado a un grupo de "
"agentes"
msgstr ""

#: api_app/tests/tests_api_call_dispositions.py:58
#: api_app/views/call_disposition.py:44
msgid "Se obtuvieron las calificaciones de forma exitosa"
msgstr ""

#: api_app/tests/tests_api_call_dispositions.py:77
#: api_app/views/call_disposition.py:113
msgid "Se creo la calificacion de forma exitosa"
msgstr ""

#: api_app/tests/tests_api_call_dispositions.py:93
#: api_app/tests/tests_external_sites.py:210
#: api_app/tests/tests_external_sites.py:234
#: api_app/views/call_disposition.py:120 api_app/views/call_disposition.py:163
#: api_app/views/external_site.py:78 api_app/views/external_site.py:112
msgid "Error al hacer la peticion"
msgstr ""

#: api_app/tests/tests_api_call_dispositions.py:113
#: api_app/views/call_disposition.py:155
msgid "Se actualizo la calificacion de forma exitosa"
msgstr ""

#: api_app/tests/tests_api_call_dispositions.py:129
#: api_app/views/call_disposition.py:193
msgid "Se elimino la calificacion de forma exitosa"
msgstr ""

#: api_app/tests/tests_api_click2call.py:105
#: api_app/tests/tests_api_click2call.py:113
#: api_app/tests/tests_api_click2call.py:121
#: api_app/tests/tests_api_click2call.py:166
msgid "Escoja una opción válida. Esa opción no está entre las disponibles."
msgstr ""

#: api_app/tests/tests_api_click2call.py:158
#: api_app/tests/tests_api_contactos.py:170
#: api_app/tests/tests_api_contactos.py:262 api_app/views/agente.py:220
#: api_app/views/base.py:87 api_app/views/base.py:215
msgid "Sistema externo inexistente."
msgstr ""

#: api_app/tests/tests_api_contactos.py:114 api_app/views/base.py:172
msgid "Contacto agregado"
msgstr ""

#: api_app/tests/tests_api_contactos.py:142
#: api_app/tests/tests_api_contactos.py:160
#: api_app/tests/tests_api_contactos.py:229
#: api_app/tests/tests_api_contactos.py:252 api_app/views/base.py:118
#: api_app/views/base.py:247
msgid "No tiene permiso para editar la campaña."
msgstr ""

#: api_app/tests/tests_api_contactos.py:178
#: api_app/tests/tests_api_contactos.py:195
#: api_app/tests/tests_api_contactos.py:272
#: api_app/tests/tests_api_contactos.py:292 api_app/views/base.py:111
#: api_app/views/base.py:240
msgid "Campaña inexistente."
msgstr ""

#: api_app/tests/tests_api_contactos.py:186
#: api_app/tests/tests_api_contactos.py:282 api_app/views/base.py:99
#: api_app/views/base.py:227
msgid "Debe indicar un idCampaign válido."
msgstr ""

#: api_app/tests/tests_api_contactos.py:302
msgid "Ya existe un contacto con ese id externo en la base de datos"
msgstr ""

#: api_app/tests/tests_api_contactos.py:334 api_app/views/base.py:135
msgid "El campo es obligatorio"
msgstr ""

#: api_app/tests/tests_api_contactos.py:345 api_app/views/base.py:127
msgid "Se recibieron campos incorrectos"
msgstr ""

#: api_app/tests/tests_api_permisos.py:56 api_app/views/administrador.py:83
msgid "Ya existe un rol con ese nombre"
msgstr ""

#: api_app/tests/tests_api_permisos.py:66 api_app/views/administrador.py:78
msgid "Se esperaba el campo \"name\""
msgstr ""

#: api_app/tests/tests_api_permisos.py:96 api_app/views/administrador.py:108
msgid "Se esperaba el campo \"role_id\" (numérico)"
msgstr ""

#: api_app/tests/tests_api_permisos.py:103
#: api_app/tests/tests_api_permisos.py:111
#: api_app/tests/tests_api_permisos.py:171 api_app/views/administrador.py:115
#: api_app/views/administrador.py:145
msgid "Id de Rol incorrecto"
msgstr ""

#: api_app/tests/tests_api_permisos.py:120
#: api_app/tests/tests_api_permisos.py:129 api_app/views/administrador.py:119
msgid "Se esperaba el campo \"permissions\" (lista)"
msgstr ""

#: api_app/tests/tests_api_permisos.py:139 api_app/views/administrador.py:124
msgid "Lista de permisos incorrecta"
msgstr ""

#: api_app/tests/tests_api_permisos.py:162 api_app/views/administrador.py:150
msgid "No se puede borrar un rol asignado a usuarios."
msgstr ""

#: api_app/tests/tests_api_subir_base_contactos.py:101
#: api_app/views/administrador.py:192
msgid "falta parámetro filename en request"
msgstr ""

#: api_app/tests/tests_api_subir_base_contactos.py:113
#: api_app/tests/tests_api_subir_base_contactos.py:125
#: api_app/views/administrador.py:214
msgid "falta parámetro:"
msgstr ""

#: api_app/tests/tests_api_subir_base_contactos.py:138
#: api_app/views/administrador.py:229
msgid "campo de teléfono no coincide con nombre de columna"
msgstr ""

#: api_app/tests/tests_api_subir_base_contactos.py:151
#: api_app/views/administrador.py:237
msgid "campo de id externo no coincide con nombre de columna"
msgstr ""

#: api_app/tests/tests_api_subir_base_contactos.py:163
#: api_app/views/administrador.py:189
msgid "la extensión del archivo no es .CSV"
msgstr ""

#: api_app/tests/tests_api_subir_base_contactos.py:177
#: api_app/views/administrador.py:185
#, python-brace-format
msgid "La longitud del nombre no debe exceder los {0} caracteres"
msgstr ""

#: api_app/tests/tests_external_sites.py:66 api_app/views/external_site.py:43
msgid "Se obtuvieron los sitios externos de forma exitosa"
msgstr ""

#: api_app/tests/tests_external_sites.py:79 api_app/views/external_site.py:140
msgid "Se obtuvo la informacion del sitio externo de forma exitosa"
msgstr ""

#: api_app/tests/tests_external_sites.py:107 api_app/views/external_site.py:165
msgid "Se elimino el sitio externo de forma exitosa"
msgstr ""

#: api_app/tests/tests_external_sites.py:123 api_app/views/external_site.py:171
msgid "No está permitido eliminar un sitio externo asociado a una campaña"
msgstr ""

#: api_app/tests/tests_external_sites.py:139 api_app/views/external_site.py:199
msgid "Se oculto el sitio externo de forma exitosa"
msgstr ""

#: api_app/tests/tests_external_sites.py:155 api_app/views/external_site.py:222
msgid "Se desoculto el sitio externo de forma exitosa"
msgstr ""

#: api_app/tests/tests_external_sites.py:179 api_app/views/external_site.py:70
msgid "Se creo el sitio externo de forma exitosa"
msgstr ""

#: api_app/views/administrador.py:202
msgid "no se pudo crear la base de datos de contacto"
msgstr ""

#: api_app/views/administrador.py:226
msgid "lista de campos teléfono vacia"
msgstr ""

#: api_app/views/administrador.py:281
#, python-brace-format
msgid "Error en el intento de conexion a: {0} debido {1}"
msgstr ""

#: api_app/views/administrador.py:286
msgid "Error en el servidor externo"
msgstr ""

#: api_app/views/agente.py:67
msgid "Error al generar sip password"
msgstr ""

#: api_app/views/agente.py:219 api_app/views/agente.py:254
#: api_app/views/base.py:76 api_app/views/base.py:204
msgid "Hubo errores en los datos recibidos"
msgstr ""

#: api_app/views/agente.py:245
msgid "Error al ejecutar la llamada"
msgstr ""

#: api_app/views/agente.py:435
msgid "Auditoría inexistente"
msgstr ""

#: api_app/views/agente.py:440
msgid "No tiene permiso para modificar la auditoría"
msgstr ""

#: api_app/views/call_disposition.py:55
msgid "Error al obtener las calificaciones"
msgstr ""

#: api_app/views/call_disposition.py:70
msgid "Se obtuvo el detalle de la calificacion de forma exitosa"
msgstr ""

#: api_app/views/call_disposition.py:80 api_app/views/call_disposition.py:209
msgid "No existe la calificacion"
msgstr ""

#: api_app/views/call_disposition.py:85
msgid "Error al obtener el detalle de la calificacion"
msgstr ""

#: api_app/views/call_disposition.py:127
msgid "Error al crear la calificacion"
msgstr ""

#: api_app/views/call_disposition.py:171
msgid "No existe la calificacion que se quiere actualizar"
msgstr ""

#: api_app/views/call_disposition.py:177
msgid "Error al actualizar la calificacion"
msgstr ""

#: api_app/views/call_disposition.py:201
msgid "No está permitido eliminar una calificacion reservada del sistema"
msgstr ""

#: api_app/views/call_disposition.py:214
msgid "Error al eliminar la calificacion"
msgstr ""

#: api_app/views/external_site.py:53
msgid "Error al obtener los sitios externos"
msgstr ""

#: api_app/views/external_site.py:84
msgid "Error al crear el sitio externo"
msgstr ""

#: api_app/views/external_site.py:101
msgid "Se actualizo el sitio externo de forma exitosa"
msgstr ""

#: api_app/views/external_site.py:118
msgid "No existe el sitio externo que se quiere actualizar"
msgstr ""

#: api_app/views/external_site.py:124
msgid "Error al actualizar el sitio externo"
msgstr ""

#: api_app/views/external_site.py:149
msgid "Error al obtener el detalle del sitio externo"
msgstr ""

#: api_app/views/external_site.py:179
msgid "No existe el sitio externo"
msgstr ""

#: api_app/views/external_site.py:184
msgid "Error al eliminar el sitio externo"
msgstr ""

#: api_app/views/external_site.py:207
msgid "Error al ocultar el sitio externo"
msgstr ""

#: api_app/views/external_site.py:230
msgid "Error al desocultar el sitio externo"
msgstr ""

#: api_app/views/grabaciones.py:91
msgid "Exportación de grabaciones Zip en proceso"
msgstr ""

#: api_app/views/supervisor.py:120 api_app/views/supervisor.py:124
#: api_app/views/supervisor.py:127
msgid "Filtro \"type\" inválido"
msgstr ""

#: api_app/views/supervisor.py:136
msgid "Filtro \"agent\" inválido"
msgstr ""

#: api_app/views/supervisor.py:140
msgid "Agente inexistente"
msgstr ""

#: api_app/views/supervisor.py:151 api_app/views/supervisor.py:156
#: api_app/views/supervisor.py:159
msgid "Filtro \"status\" inválido"
msgstr ""

#: api_app/views/supervisor.py:312 api_app/views/supervisor.py:358
msgid "ID Agenda incorrecto"
msgstr ""

#: api_app/views/supervisor.py:319
msgid "ID Agente incorrecto"
msgstr ""

#: api_app/views/supervisor.py:327 api_app/views/supervisor.py:365
msgid "No tiene permiso para editar esta Agenda"
msgstr ""

#: api_app/views/supervisor.py:384
msgid "Recibidas"
msgstr ""

#: api_app/views/supervisor.py:385
msgid "Efectuadas"
msgstr ""

#: api_app/views/supervisor.py:386
msgid "Atendidas"
msgstr ""

#: api_app/views/supervisor.py:387
msgid "Conectadas"
msgstr ""

#: api_app/views/supervisor.py:388
msgid "No Conectadas"
msgstr ""

#: api_app/views/supervisor.py:389
msgid "Abandonadas"
msgstr ""

#: api_app/views/supervisor.py:390
msgid "Expiradas"
msgstr ""

#: api_app/views/supervisor.py:391
msgid "Tiempo de Espera de Conexión(prom.)"
msgstr ""

#: api_app/views/supervisor.py:392
msgid "Tiempo de Espera de Atención(prom.)"
msgstr ""

#: api_app/views/supervisor.py:393
msgid "Tiempo de Abandono(prom.)"
msgstr ""

#: api_app/views/supervisor.py:396
msgid "Efectuadas Manuales"
msgstr ""

#: api_app/views/supervisor.py:397
msgid "Conectadas Manuales"
msgstr ""

#: api_app/views/supervisor.py:398
msgid "No Conectadas Manuales"
msgstr ""

#: api_app/views/supervisor.py:399
msgid "Tiempo de Espera de Conexión Manuales(prom.)"
msgstr ""

#: api_app/views/supervisor.py:419 api_app/views/supervisor.py:434
#: api_app/views/supervisor.py:940
msgid "No existe la campaña"
msgstr ""

#: api_app/views/supervisor.py:525
msgid "Exportación de CSV en proceso"
msgstr ""

#: api_app/views/supervisor.py:573
msgid "Exportación de contactados a .csv en proceso"
msgstr ""

#: api_app/views/supervisor.py:620
msgid "Exportación de calificados a .csv en proceso"
msgstr ""

#: api_app/views/supervisor.py:667
msgid "Exportación de no atendidos a .csv en proceso"
msgstr ""

#: api_app/views/supervisor.py:679
msgid "Finalizado"
msgstr ""

#: api_app/views/supervisor.py:681
msgid "Liberado"
msgstr ""

#: api_app/views/supervisor.py:683
msgid "Reservado"
msgstr ""

#: api_app/views/supervisor.py:767
msgid "Exportación de calificaciones a .csv en proceso"
msgstr ""

#: api_app/views/supervisor.py:812
msgid "Exportación de Formularios de Gestiones a .csv en proceso"
msgstr ""

#: api_app/views/supervisor.py:936
msgid "La campaña no está activa"
msgstr ""

#: api_app/views/supervisor.py:964
#, python-brace-format
msgid "QueueAdd failed - agente: {0} de la campana: {1} "
msgstr ""

#: api_app/views/supervisor.py:989
#, python-brace-format
msgid "QueueRemove failed - agente: {0} de la campana: {1}"
msgstr ""

#: api_app/views/supervisor.py:1000
msgid "No existe el agente en cola"
msgstr ""

#: api_app/views/supervisor.py:1022
msgid "No existe la campaña, no se pueden agregar agentes"
msgstr ""

#: api_app/views/supervisor.py:1039
msgid ""
"No existe el agente,                                            no se puede "
"eliminar de la member queue"
msgstr ""

#: api_app/views/supervisor.py:1050
msgid ""
"No existe el agente,                                            no se puede "
"crear la member queue"
msgstr ""

#: api_app/views/supervisor.py:1068
msgid "No se pudo confirmar la creación del dialplan"
msgstr ""

#: api_app/views/supervisor.py:1072
msgid "La campaña no está activa, no se pueden agregar agentes"
msgstr ""

#: api_app/views/supervisor.py:1104
msgid "Error al obtener los agentes activos"
msgstr ""

#: api_app/views/supervisor.py:1118
msgid "Se obtuvieron las pausas de forma exitosa"
msgstr ""

#: api_app/views/supervisor.py:1127
msgid "Error al obtener las pausas"
msgstr ""

#: api_app/views/supervisor.py:1152
msgid "Error al obtener los conjuntos de pausas"
msgstr ""

#: api_app/views/supervisor.py:1167
msgid "Se obtuvo la informacion del conjunto de pausa de forma exitosa"
msgstr ""

#: api_app/views/supervisor.py:1184
msgid "Error al obtener el detalle del conjunto de pausa"
msgstr ""

#: api_app/views/supervisor.py:1222
msgid "Error al crear el conjunto de pausa"
msgstr ""

#: api_app/views/supervisor.py:1250 api_app/views/supervisor.py:1284
#: api_app/views/supervisor.py:1326
msgid "No existe el conjunto de pausa"
msgstr ""

#: api_app/views/supervisor.py:1255
msgid "Error al actualizar el conjunto de pausa"
msgstr ""

#: api_app/views/supervisor.py:1289
msgid "Error al eliminar el conjunto de pausa"
msgstr ""

#: api_app/views/supervisor.py:1318
msgid "No existe la pausa"
msgstr ""

#: api_app/views/supervisor.py:1336
msgid "Error al crear la configuracion de pausa"
msgstr ""

#: api_app/views/supervisor.py:1364 api_app/views/supervisor.py:1394
msgid "No existe la configuracion de pausa"
msgstr ""

#: api_app/views/supervisor.py:1369
msgid "Error al eliminar la configuracion de pausa"
msgstr ""

#: api_app/views/supervisor.py:1399
msgid "Error al actualizar la configuracion de pausa"
msgstr ""

#: api_app/views/wombat_dialer.py:48 api_app/views/wombat_dialer.py:92
msgid "Refrescando. Espere al menos 15 segundos mientras finaliza el proceso."
msgstr ""

#: utiles_globales.py:37
msgid "Archivos permitidos: .wav"
msgstr ""
